virtio-queue/src/queue/verification.rs

```diff
@@ -0,0 +1,271 @@
+
```

Comment:
@priyasiddharth remove this line and rebase, thanks!

Reply:
@priyasiddharth please also resolve all the conversations that you already fixed.

Reply:
@priyasiddharth I re-opened this, because I saw it wasn't rebased (please check it at the end of the PR, there is a big message telling that we can't merge, because needs to be rebased, so if you see it, please rebase).
BTW for now I did the rebase since there were no conflicts.